conntact/ReadMe.md

```diff
@@ -315,6 +369,22 @@ To run the example, open a terminals sourced to the built project workspace and
 
     roslaunch conntact conntact_demo.launch
 
+## Appendix
+
+###### Interpolation
+
+Our team has implemented a solution to the potential danger of the CCC's setpoint-seeking PD loop. We find that, if the robot is sent to a distant command, the forces and speeds used by the robot to reach that command pose a serious collision danger. It's possible that some limits/parameters can be set in CCC directly to control this, but we didn't find them and needed a quick and certain fix to the rapid unplanned motion problem.
+
+In Conntext's `publish_pose` method, we run the command from the Conntask through an interpolation function `assembly_tools/` which limits its *distance in position and orientation* to a user-specified cap.
```

Comment:
I believe `assembly_tools/` no longer exists. Should it be "an interpolation function in `assembly_utils.py`"?

Reply:
Thanks, good catch. Fixed it.